boards/nxp/frdm_ke15z/frdm_ke15z.dts

```diff
@@ -0,0 +1,216 @@
+		zephyr,uart-pipe = &lpuart1;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		red_led: led_0 {
+			gpios = <&gpiod 0 GPIO_ACTIVE_LOW>;
+			label = "User LD1";
+		};
+		green_led: led_1 {
+			gpios = <&gpiod 16 GPIO_ACTIVE_LOW>;
+			label = "User LD2";
+		};
+		blue_led: led_2 {
+			gpios = <&gpiod 15 GPIO_ACTIVE_LOW>;
+			label = "User LD3";
+		};
+	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+		red_pwm_led: led_pwm_0 {
+			pwms = <&ftm0 2 PWM_MSEC(20) PWM_POLARITY_INVERTED>;
+			label = "Red RGB LED D4";
+		};
+		yellow_pwm_led: led_pwm_1 {
+			pwms = <&ftm0 1 PWM_MSEC(20) PWM_POLARITY_INVERTED>;
+			label = "Green RGB LED D4";
+		};
+		green_pwm_led: led_pwm_2 {
+			pwms = <&ftm0 0 PWM_MSEC(20) PWM_POLARITY_INVERTED>;
+			label = "Blue RGB LED D4";
+		};
+
+	};
+
+
+
+
+	gpio_keys {
+		compatible = "gpio-keys";
+		user_button_0: button_0 {
+			label = "User SW3";
+			gpios = <&gpiod 3 GPIO_ACTIVE_LOW>;
+			zephyr,code = <INPUT_KEY_0>;
+		};
+		user_button_1: button_1 {
+			label = "User SW2";
+			gpios = <&gpiob 11 GPIO_ACTIVE_LOW>;
+			zephyr,code = <INPUT_KEY_1>;
+		};
+	};
+};
+
+&cpu0 {
+	clock-frequency = <120000000>;
+};
+
+/*  b01800 Check Low power mode */
+&idle {
+	min-residency-us = <1>;
+};
+
+
+&stop {
+	min-residency-us = <20000>;
+	exit-latency-us = <13>;
+};
+
+
+&sim {
+	clkout-source = <1>;
+	clkout-divider = <0>;
+};
+
+&scg {
+	sosc-mode = <KINETIS_SCG_SOSC_MODE_LOW_POWER>;
+
+	sosc_clk {
+		status = "okay";
+		clock-frequency = <8000000>;
+	};
+
+	core_clk {
+		clocks = <&lpfll_clk>;
+	};
+
+	core_clk {
+		clocks = <&firc_clk>;
+	};
+
+	bus_clk {
+		clock-div = <2>;
+	};
+
+	slow_clk {
+		clock-div = <5>;
+	};
+
+	clkout_clk {
+		status = "okay";
+	};
+
+	flldiv2_clk {
+		clock-div = <2>;
+	};
+
+	sircdiv1_clk {
+		clock-div = <1>;
+	};
+
+	sircdiv2_clk {
+		clock-div = <2>;
+	};
+
+	fircdiv1_clk {
+		clock-div = <1>;
+	};
+
+	fircdiv2_clk {
+		clock-div = <1>;
+	};
+
+	soscdiv1_clk {
+		clock-div = <1>;
+	};
+
+	soscdiv2_clk {
+		clock-div = <1>;
+	};
+};
+
+&lpuart1 {
+	status = "okay";
+	current-speed = <115200>;
+	pinctrl-0 = <&lpuart1_default>;
+	pinctrl-names = "default";
+};
+
+&ftm0 {
+	status = "okay";
+	compatible = "nxp,kinetis-ftm-pwm";
+	#pwm-cells = <3>;
+	pinctrl-0 = <&ftm0_default>;
+	pinctrl-names = "default";
+};
+&lpi2c0 {
+	status = "okay";
+	pinctrl-0 = <&lpi2c0_default>;
+	pinctrl-names = "default";
+	scl-gpios = <&gpioa 3 GPIO_ACTIVE_HIGH>;
+	sda-gpios = <&gpioa 2 GPIO_ACTIVE_HIGH>;
+
+	fxos8700: fxos8700@1d {
+		compatible = "nxp,fxos8700";
+		reg = <0x1d>;
+		reset-gpios = <&gpiob 15 GPIO_ACTIVE_HIGH>;
+	};
+};
+
+&lpspi0 {
+	dmas = <&edma 0 14>, <&edma 1 15>;
+	dma-names = "rx", "tx";
+	status = "okay";
+	pinctrl-0 = <&lpspi0_default>;
+	pinctrl-names = "default";
+};
+
+/*
+ * &lpspi1 {
+ *	status = "okay";
+ *	pinctrl-0 = <&lpspi1_default>;
+ *	pinctrl-names = "default";
+ *};
+ */
+
+&gpiob {
+	status = "okay";
+};
+
+&gpiod {
+	status = "okay";
+};
+
+&flexio {
+	status = "okay";
```

Comment:
There are a lot of feature enabled here. Has everything been tested?

Reply:
I have started testing